eng/pipelines/templates/steps/build-package-artifacts.yml

```diff
@@ -33,6 +34,13 @@ steps:
       pip install wheel setuptools pathlib twine readme-renderer[md] packaging
     displayName: 'Prep Environment'
 
+  - task: PythonScript@0
```

Comment:
Assuming this is the only template that produces packages I think it is the only place we need to set the dev version.